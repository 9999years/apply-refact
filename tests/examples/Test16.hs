concatMop f x = concat (map f x)