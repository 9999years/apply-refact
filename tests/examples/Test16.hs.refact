[("tests/examples/Test16.hs:1:17: Error: Use concatMap\nFound:\n  concat (map f x)\nWhy not:\n  concatMap f x\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 17, endLine = 1, endCol = 33}, subts = [("f",SrcSpan {startLine = 1, startCol = 29, endLine = 1, endCol = 30}),("x",SrcSpan {startLine = 1, startCol = 31, endLine = 1, endCol = 32})], orig = "concatMap f x"}])]
