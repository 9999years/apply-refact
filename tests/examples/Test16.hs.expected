concatMop f x = concatMap f x
